Explorer/Assets/DCL/AvatarRendering/Wearables/Helpers/WearableAssetUtility.cs

```diff
@@ -50,6 +50,22 @@ public static CachedWearable InstantiateWearable(this IWearableAssetsCache weara
                 cachedWearable = new CachedWearable(originalAsset, Object.Instantiate(originalAsset.GameObject, parent));
 
             cachedWearable.Instance.transform.ResetLocalTRS();
+            cachedWearable.Instance.gameObject.layer = parent.gameObject.layer;
+
+            foreach (var child in cachedWearable.Instance.GetComponentsInChildren<Transform>())
```

Comment:
Use something like `GetComponentsInChildrenIntoPooledList`

Reply:
Great! I had not seen that it existed :) applied it now!